src/finitediff/timedomain/recordings.jl

```diff
@@ -22,7 +22,29 @@ struct Recordings{Ti<:Int64, Tv<:AbstractFloat}
 end
 
 """
-   constructor for recordings
+
+R = Recordings(rz, rx, params; <keyword arguments>)
```

Comment:
I like it! Just give four spaces before start writing R = ...

Reply:
Ok!